README.md

```diff
@@ -8,6 +8,24 @@ The cookbook includes a thorfile to encrypt your secrets into .crypt files, and
 
 Additionally the readme is going to advise you which IAM roles you should assign an instance to run this without keys.
 
+# The idea
+
+Managing secrets is hard. Chef encrypted data bags are not ideal, and are reasonably hard to manage. Chef vault has overheads... So this is where KMS steps in.
+
+KMS enables you to create a private key in amazon, and control access to it. You can then call the API to encrypt and decrypt data, if you have access privileges to that key.
+
+Therefore, you can:
+  - Encrypt all your data into files using a key
+  - Bake the files into your amazon AMIs (or drop them off) and make sure they are owned by root...
```

Comment:
by drop them off do you mean pull them from a remote source like s3? Some clarification would be nice here.